tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -246,6 +246,24 @@ Verify "Enabled" Keeps Being Available After One Of The ISV Operators If Uninsta
    Uninstall Operator And Check Enabled Page Is Rendering  operator_name=${openvino_operator_name}  operator_appname=${openvino_appname}
    [Teardown]    Check And Uninstall Operator In Openshift    ${openvino_operator_name}   ${openvino_appname}
 
+Verify external links in Quick Starts are not broken
+        [Tags]    Tier1
+        ...       ODS-1305
+        [Documentation]    Verify external links in Quick Starts are not broken
+        Click Link    Resources
+        Wait Until Resource Page Is Loaded
+        Wait Until Element Is Visible   xpath=//a[contains(@class,'odh-card__footer__link') and contains(@href,'#')]
+        ${qucickStartElements}=   Get WebElements    xpath=//a[contains(@class,'odh-card__footer__link') and contains(@href,'#')]
```

Comment:
could you also print the name or the id of the quick start? this would make easier to read and debug the logs